packages/compartment-mapper/test/test-cjs-compat.js

```diff
@@ -0,0 +1,42 @@
+// import "./ses-lockdown.js";
+import '../../ses/index.js';
+import fs from 'fs';
+import url from 'url';
+import test from 'ava';
+import { loadLocation } from '../src/import.js';
+import { makeReadPowers } from '../src/node-powers.js';
+
+const readPowers = makeReadPowers({ fs, url });
+const { read } = readPowers;
+
+test('CommonJS referencing exports', async t => {
+  t.plan(1);
+
+  const fixture = new URL(
+    'fixtures-cjs-compat/node_modules/app/index.js',
+    import.meta.url,
+  ).toString();
+
+  const application = await loadLocation(read, fixture);
+  const {
+    namespace: { assertions },
+  } = await application.import({});
+  assertions.packageReferencingItself();
```

Comment:
:question: Would you rather see the assertions made inside the test itself? 

I didn't want to go through another layer of exporting and importing the same thing before checking, so I implemented cheching inside fixtures ([there's precedent!](https://github.com/endojs/endo/blob/3bbe1802cb789c975f3e190e234c061b6613742b/packages/compartment-mapper/test/fixtures-0/node_modules/app/main.js))

Reply:
This is fine.

Reply:
For the back of the mind: it may begin to make sense to write these tests as test262 fixtures instead.